halo1/model.py

```diff
@@ -334,9 +344,40 @@ def import_halo1_region_from_jms(jms, *, name="unnamed", scale=1.0, region_filte
 	scene = bpy.context.collection
 	scene.objects.link(region_obj)
 
+	# If the function was supplied with an armature object attempt to skin to it.
+
+	region_obj.parent = armature_obj
+
+	if skin_vertices and region_obj.parent.type == 'ARMATURE':
+		mod = region_obj.modifiers.new('armature', 'ARMATURE')
+		mod.object = armature_obj
+
+		# Create a vertex group for each bone.
+
+		for node in jms.nodes:
+			region_obj.vertex_groups.new(name=NODE_NAME_PREFIX+node.name)
+
+		vertex_groups = region_obj.vertex_groups
+
+		# Add the vertices to all the correct vertex groups.
+
+		for jms_i in translation_dict:
+			v = jms.verts[jms_i]
+			mesh_i = translation_dict[jms_i]
+
+			if v.node_0 != -1:
```

Comment:
What is the significance of -1 in this context?

Reply:
-1 basically is Halo's None. Before this function I've been checking if those values are "in" range which automatically culled that out. If we made this a constant, what would we name it you think?

Reply:
`HALO_NONE`, probably.